recipe/meta.yaml

```diff
@@ -10,8 +10,6 @@ source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256sum }}
-  patches:
-    - patches/0001-Fix-integer-check.patch
 
 build:
   number: 0
```

Comment:
I'd rather fix in upstream and make a 6.6.1. I'll think about how to do this.

Thinc doesn't require CUDA -- it should degrade to CPU if it's not available. It's just a question of getting the setup.py logic right.